### ISSUE TITLE
[BUG] 

### ISSUE THREAD

**Issue by invalidgdash** [bedrock-viz/bedrock-viz#109]
the bedrock-viz.exe doesn't work win10

**Comment by counterfitninja** [comment]
Can you provide more information than it doesn't work? Does it run at all? what errors if any do you get?

**Comment by Plidian** [comment]
Just to provide some additional information, I was able to download the latest build of bedrock-viz on windows 10 2004 and ran it successfully from a command line so it is not universally un-usable.

**Comment by cabbey** [comment]
yeah, going to need a lot more details to help with this, but the first thing I'm going to have to ask is: did you read the directions? this is a command line tool, no GUI.
